### PR TITLE
Tribe ext selectively show html

### PR DESCRIPTION
**Pull request by elimn**
Adds fields to WP Admin > Events > Settings > Display for selecting which views to see the HTML Before/After content on. This allows you to show these boxes on views of your choosing.

### CODE REVIEW

File: tribe-ext-selectively-show-html/index.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+		require_once dirname( __FILE__ ) . '/src/functions/general.php';
+		require_once dirname( __FILE__ ) . '/src/Tribe/View_Helper.php';
+
+		add_action( 'admin_init', array( $this, 'add_settings' ) );
+
+		add_filter( 'tribe_events_before_html', array( $this, 'tribe_events_before_html' ) );
+		add_filter( 'tribe_events_after_html', array( $this, 'tribe_events_after_html' ) );
+	}
+
+	public function add_settings() {
```
Comment (ggwicz) [inline tribe-ext-selectively-show-html/index.php:55]: no docblock? i'm fine without it, but just wanted to highlight its absence

File: tribe-ext-selectively-show-html/index.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+		// Setup fields on the settings page
+		$setting_helper = new Tribe__Settings_Helper();
+
+		$setting_helper->add_field(
+			'show_html_before_views',
+			array(
+				'type'            => 'checkbox_list',
+				'label'           => __( 'Show HTML before on', 'tribe-extension' ),
+				'tooltip'         => __( 'Show the HTML before content on these views', 'tribe-extension' ),
```
Comment (ggwicz) [inline tribe-ext-selectively-show-html/index.php]: for this line (`70`) and above: no `esc_html__()`?

File: tribe-ext-selectively-show-html/index.php [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+			'tribeEventsBeforeHTML',
+			false
+		);
+
+		$setting_helper->add_field(
+			'show_html_after_views',
+			array(
+				'type'            => 'checkbox_list',
+				'label'           => __( 'Show HTML after on', 'tribe-extension' ),
+				'tooltip'         => __( 'Show the HTML after content on these views', 'tribe-extension' ),
```
Comment (ggwicz) [inline tribe-ext-selectively-show-html/index.php]: for this line (`85`) and above: no `esc_html__()`?

File: tribe-ext-selectively-show-html/src/functions/general.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
+<?php
```
Comment (ggwicz) [inline tribe-ext-selectively-show-html/src/functions/general.php:1]: I'm a little confused about this existence of this file — `/src/functions/general.php` — and the above file, `/src/Tribe/functions/general.php`

The latter seems to basically just be the *exact duplicate* of the former, with the addition of `tribe_call_func()` and `tribe_is_ajax()`

Is there some reason these two files to both exist? If not, one should be deleted, it seems...

**Review by ggwicz — CHANGES_REQUESTED** [review]

File: tribe-ext-selectively-show-html/src/functions/general.php (hunk 4 above)
Comment (elimn) [reply tribe-ext-selectively-show-html/src/functions/general.php:1]: The one with tribe_call_func() was a relic from the snippets framework. This used to be built atop that. Thanks for pointing this out, that file is now deleted.

**Review by ggwicz — APPROVED** [review]

**Pull request merged**